spec/pubid_iso/identifier_spec.rb

```diff
@@ -735,5 +735,13 @@
         end
       end
     end
+
```

Comment:
Can we add a test for default publisher? Thanks.

Reply:
> Can we add a test for default publisher? Thanks.

It's here already https://github.com/metanorma/pubid-iso/pull/86/files/5b135efca112aa562ab29de8d8a30ec6657c0932#diff-59800c3156cfaca079c88759e3ef3df7589c552a11991aa5270988ee28d0a57bL705

Reply:
> Can we add a test for default publisher? Thanks.

It's on line 705:
<img width="794" alt="image" src="https://user-images.githubusercontent.com/5849/187669064-dc2076d6-b244-4549-96c3-f9aaa9b4ed2f.png">

Reply:
But that is really about IDF, I want a test that only tests "default publisher".

Reply:
A unit test is supposed to test a singular function on a single case. I prefer not using composite tests to cover multiple features at once, those are supposed to test "integration".